discord/guild.py

```diff
@@ -1018,6 +1021,11 @@ def created_at(self) -> datetime.datetime:
         """:class:`datetime.datetime`: Returns the guild's creation time in UTC."""
         return utils.snowflake_time(self.id)
 
+    @property
+    def invites_disabled(self) -> bool:
+        """:class:`bool`: Returns a boolean indicating if the guild invites are disabled"""
```

Comment:
```suggestion
        """:class:`bool`: Returns a boolean indicating if the guild invites are disabled."""
```

Very small thing (as in a literal .), but we just want to be consistent with the rest.